lib/UtapiClient.js

```diff
@@ -264,6 +275,40 @@ class UtapiClient {
     }
 
     /**
+    * Publish an event in the bucket's dedicated channel to notify subscribers
+    * for performed actions with side effects
+    * @param {string} metric - metric to be published
+    * @param {object} params - params for the metrics
+    * @param {string} [params.bucket] - (optional) bucket name
+    * @param {array} [params.objects] - (optional) objects affected
+    * @param {object} log - logger
+    * @param {callback} [callback] - (optional) callback to call
+    * @return {undefined}
+    */
+    _publishEvent(metric, params, log, callback) {
+        const { bucket, objects } = params;
+        let message = metric;
+        if (Array.isArray(objects)) {
+            objects.forEach(object => {
+                message = `${message}:${object}`;
+            });
+        }
+        return this.ds.publish(`s3:event:${bucket}`, message, err => {
+            if (err) {
+                log.error('Failed to publish event, ignored', {
+                    bucket,
+                    message,
+                    error: err,
+                });
+            }
+            if (typeof callback === 'function') {
```

Comment:
We already have this check in the calling function, so you can call the callback directly.

Reply:
that's not the case with the command ```npm run ft_test```